packages/client/src/components/panels/InvitePanel/sub-components/Item.js

```diff
@@ -110,9 +111,9 @@ const Item = ({
 
   const displayBody = (
     <>
-      <Text {...textProps} noSelect>
+      <StyledText {...textProps} noSelect>
```

Comment:
Just use 'truncate' prop for <Text /> component

Reply:
use 'truncate' prop